easybuild/easyconfigs/m/makedepf90/makedepf90-2.8.8-foss-2017a.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'makedepf90'
+version = '2.8.8'
+
+homepage = ''
+description = """ Makedepf90 is a program for automatic creation of
+ Makefile-style dependency lists for Fortran source code. """
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+sources = ['%(name)s_%(version)s.orig.tar.gz']
+source_urls = ['https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/%(name)s/%(version)s-1/']
```

Comment:
@dplacencia Please include a `SHA256` checksum, `eb makedepf90-2.8.8-foss-2017a.eb --inject-checksums` will do this for you